docs/recipes/asset-exchange.md

```diff
@@ -0,0 +1,52 @@
+# Asset Exchange
+
+Distinct [`ledger`](../reference/accounts.md#ledger) values typically denote distinct assets
```

Comment:
Should we explain what an asset change is first before diving into how it's implemented in TigerBeetle?